styles.css

```diff
@@ -1,6 +1,11 @@
-header a {
-  text-decoration: none;
+body {
+  background-image: url(../crispy-couscous/crispy-images/pexel-photo.jpeg);
```

Comment:
this url should not need the repo name in it... it should just be relative to the css directory.